lib/get_parser.py

```diff
@@ -234,6 +234,18 @@ def get_parser() -> argparse.ArgumentParser:
         default=[""],
         help="Lines to append to the job file.",
     )
+    parser.add_argument(
+        "--need-modify",
```

Comment:
I realized in my writeup, I called this `--need-modify`, but should we actually make that `--need-storage-modify` since there are modify scopes for other types (like `compute.modify`)?  What do you think?

Reply:
Making this change...

Reply:
This is in now.